File: PHP/level-400.php
<?php
    for($i = 1; $i <= 100; $i++) {
        if (is_fizz_buzz($i)) {
            echo 'FizzBuzz', "\n";
        } else if (is_fizz($i)) {
            echo 'Fizz', "\n";
        } else if (is_buzz($i)) {
            echo 'Buzz', "\n";
        } else {
            echo $i, "\n";
        }
    }

    function is_fizz_buzz($number) {
        return is_fizz($number) && is_buzz($number);
    }

    function is_fizz($number) {
        return $number % 3 == 0;
    }

    function is_buzz($number) {
        return $number % 5 == 0;
    }
?>

File: Java/level-400.java
public class Main {
    public static void main(String[] args) {
        for (int i = 1; i <= 100; i++) {
            if (isFizzBuzz(i)) {
                System.out.println("FizzBuzz");
            } else if (isFizz(i)) {
                System.out.println("Fizz");
            } else if (isBuzz(i)) {
                System.out.println("Buzz");
            } else {
                System.out.println(i);
            }
        }
    }

    private static boolean isFizzBuzz(int number) {
        return isFizz(number) && isBuzz(number);
    }

    private static boolean isFizz(int number) {
        return number % 3 == 0;
    }

    private static boolean isBuzz(int number) {
        return number % 5 == 0;
    }
}
